src/main/scala/sbtdatabricks/DatabricksPlugin.scala

```diff
@@ -183,3 +237,9 @@ case class LibraryStatus(
     attachAllClusters: Boolean,
     statuses: List[LibraryClusterStatus])
 case class LibraryClusterStatus(clusterId: String, status: String)
+case class ContextId(id: String)
+case class ContextStatus(status: String, id: String)
```

Comment:
@MasseGuillaume this is the return type of the Commands. I've ripped out the comparisons to these responses to an ADT.